src/components/Footer.vue

```diff
@@ -4,13 +4,13 @@ import { RouterLink } from 'vue-router'
 
 <template>
   <div
-    class="flex justify-between items-center drop-shadow-[0_-4px_3px_rgba(0,0,0,0.25)] bg-slate text-gray-50 py-1 px-3"
+    class="flex justify-between items-center drop-shadow-[0_-4px_3px_rgba(0,0,0,0.25)] bg-slate text-gray-50 py-6 lg:py-1 px-3"
   >
-    <a href="https://ubcbiomod.com" class="pl-2">
+    <a href="https://ubcbiomod.com" class="lg:pl-2 lg:text-lg text-center">
       See Past <span class="font-semibold">UBC Biomod</span> Projects
     </a>
 
-    <RouterLink to="/">
+    <RouterLink to="/" class="hidden lg:block">
```

Comment:
Why are we hiding the "Back to homepage" Logo for responsiveness? Are you going to disable this feature completely on mobile?

Reply:
Oh nvm, I didn't see that you add this to the navbar on mobile instead. Smart guy!